package.json

```diff
@@ -25,10 +25,25 @@
   },
   "homepage": "https://github.com/AutolabJS/autolabcli#readme",
   "dependencies": {
+    "@input/eval": "file:lib/cli/input/eval",
+    "@input/init": "file:lib/cli/input/init",
+    "@input/prefs": "file:lib/cli/input/prefs",
+    "@model/eval": "file:lib/model/eval",
+    "@model/exit": "file:lib/model/exit",
+    "@model/init": "file:lib/model/init",
+    "@model/prefs": "file:lib/model/prefs",
+    "@output/eval": "file:lib/cli/output/eval",
+    "@output/exit": "file:lib/cli/output/exit",
+    "@output/init": "file:lib/cli/output/init",
+    "@output/prefs": "file:lib/cli/output/prefs",
+    "@utils/PromptGenerator": "file:lib/utils/PromptGenerator",
+    "@utils/command-validator": "file:lib/utils/command-validator",
+    "@utils/preference-manager": "file:lib/utils/preference-manager",
     "caporal": "^0.9.0",
     "chalk": "^2.3.0",
     "cli-spinner": "^0.2.8",
     "cli-table": "^0.3.1",
+    "controller": "file:lib/controller",
```

Comment:
how is this used?

Reply:
It is a single dependency for all the files inside lib/controller folder. Will rename it to @controller for better readability

Reply:
To add to what @ishitamed19 said. Whenever we import the folder as a module directly, it takes `index.js` file as the entry point. Therefore, there is no need to add such a dependency on `package.json`. 
That said, it would be recommended to have an alias such as `@controller` as we import the contents of the modulle in the tests.